playground/infrastructure/ci_helper.py

```diff
@@ -42,10 +42,12 @@ async def verify_examples(self, examples: List[Example]):
 
     1. Find all beam examples starting from directory os.getenv("BEAM_ROOT_DIR")
     2. Group code of examples by their SDK.
-    3. Run processing for all examples to verify examples' code.
+    3. Run processing for single-file examples to verify examples' code.
     """
-    await get_statuses(examples)
-    await self._verify_examples(examples)
+    single_file_examples = [example for example in examples
+                            if example.tag.multifile is False]
```

Comment:
same, can we use the filter for filtering? :)

Reply:
Done.